lib/rules/property-no-unknown/__tests__/index.js

```diff
@@ -360,3 +360,54 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: [true, { ignoreAtRules: ['supports', /^my-/] }],
+
+	accept: [
+		{
+			code: '@supports(display: grid) { my-property: 1; }',
+		},
+		{
+			code: '@my-at-rule { foo: 1; }',
+		},
+		{
+			code: '@supports(display:grid) { @media (min-width: 10px) { foo: 1; } }',
+		},
+		{
+			code:
+				'@supports(display:grid) { @media (min-width: 10px) {  @my-property-value: { foo: 1; } } }',
```

Comment:
```suggestion
				'@supports(display:grid) { @media (min-width: 10px) {  a { foo: 1; } } }',
```

`@my-property-value:` is not standard at-rule. Can we use `a` instead?